### PR TITLE
Read data from json file and display it on map.

### PR DESCRIPTION
**Pull request by dexterastin**
Read data from json file and display it on map. Overall structural change.

Move the map based on the card that was moved to the middle of the card swap.
There seems to be a better algorithm or behavior, but I could not find it.
Help me! 😂

### CODE REVIEW

File: front-end/js/script.js [hunk 1]
```diff
@@ -1,64 +1,90 @@
...
+        }
+    });
+
+    cards.carousel(
+        {
+            dist: 0,
+            padding: 10,
+            fullwidth: true,
+            shift: 10
+
```
Comment (shberry) [inline front-end/js/script.js]: remove extra line :)

File: front-end/js/script.js [hunk 2]
```diff
@@ -1,64 +1,90 @@
...
+        if (cards.hasClass('initialized')) {
+            cards.removeClass('initialized')
+        }
+    });
+
+    cards.carousel(
+        {
+            dist: 0,
+            padding: 10,
+            fullwidth: true,
```
Comment (shberry) [inline front-end/js/script.js]: alphabetical order

Comment (shberry) [reply front-end/js/script.js]: same below

File: front-end/js/script.js [hunk 3]
```diff
@@ -1,64 +1,90 @@
+var places = [];
+var markers = [];
+var map;
+
```
Comment (shberry) [inline front-end/js/script.js]: Regarding the file name 'script.js' can we be more specific?

**Review by shberry — COMMENTED** [review]
Cool :)

File: front-end/js/script.js (hunk 3 above)
Comment (dexterastin) [reply front-end/js/script.js]: ok!

**Comment by dexterastin** [comment]
I modified code!! Please approve! @shberry @kmu-sv/team-alpha

File: front-end/js/initCardMapView.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+function drop() {
+    clearMarkers();
+    for (var i = 0; i < places.length; i++) {
+        addMarkerWithTimeout(places[i], i * 200, i);
+    }
+}
+
+function addMarkerWithTimeout(position, timeout, idx) {
+    window.setTimeout(function () {
+
```
Comment (khc196) [inline front-end/js/initCardMapView.js]: Remove extra line :)

**Review by khc196 — COMMENTED** [review]
Cool :)

**Review by khc196 — APPROVED** [review]

**Pull request merged**